### PR TITLE
kernel-selftests: LKP WARN miss config

### PR DESCRIPTION
**Pull request by jiaqizhao122**
fix for the following WARN info:
LKP WARN miss config CONFIG_BONDING= of net/config
	LKP WARN miss config CONFIG_GENEVE= of net/config
	LKP WARN miss config CONFIG_IPV6_SEG6_LWTUNNEL= of net/config
	LKP WARN miss config CONFIG_IP_DCCP= of net/config
	LKP WARN miss config CONFIG_IP_GRE= of net/config
	LKP WARN miss config CONFIG_MACVLAN= of net/config
	LKP WARN miss config CONFIG_MACVTAP= of net/config
	LKP WARN miss config CONFIG_XFRM_INTERFACE= of net/config

### CODE REVIEW

File: include/kernel-selftests [hunk 1]
```diff
@@ -111,6 +111,7 @@ need_kconfig:
 - IPV6_SEG6_LWTUNNEL: y
 - XFRM_INTERFACE: m
 - XFRM: y
+- BONDING: m
```
Comment (rli9) [inline include/kernel-selftests:114]: just use - BONDING without specifying m or y

File: include/kernel-selftests [hunk 2]
```diff
@@ -109,6 +109,8 @@ need_kconfig:
 - NET_CORE: y
 - UML: y
 - IPV6_SEG6_LWTUNNEL: y
+- XFRM_INTERFACE: m
```
Comment (rli9) [inline include/kernel-selftests:112]: just - XFRM_INTERFACE without m

File: include/kernel-selftests [hunk 3]
```diff
@@ -112,6 +112,7 @@ need_kconfig:
 - XFRM_INTERFACE: m
 - XFRM: y
 - BONDING: m
+- IP_DCCP: m
```
Comment (rli9) [inline include/kernel-selftests:115]: just IP_DCCP

File: include/kernel-selftests [hunk 4]
```diff
@@ -113,6 +113,7 @@ need_kconfig:
 - XFRM: y
 - BONDING: m
 - IP_DCCP: m
+- MACVLAN: m
```
Comment (rli9) [inline include/kernel-selftests:116]: just MACVLAN

File: include/kernel-selftests [hunk 5]
```diff
@@ -114,6 +114,7 @@ need_kconfig:
 - BONDING: m
 - IP_DCCP: m
 - MACVLAN: m
+- MACVTAP: m
```
Comment (rli9) [inline include/kernel-selftests:117]: also only specify MACVTAP without m

File: include/kernel-selftests (hunk 1 above)
Comment (jiaqizhao122) [reply include/kernel-selftests:114]: ok

File: include/kernel-selftests (hunk 2 above)
Comment (jiaqizhao122) [reply include/kernel-selftests:112]: ok

File: include/kernel-selftests (hunk 3 above)
Comment (jiaqizhao122) [reply include/kernel-selftests:115]: ok

File: include/kernel-selftests (hunk 4 above)
Comment (jiaqizhao122) [reply include/kernel-selftests:116]: ok

**Comment by rli9** [comment]
thanks for the patch

**Pull request merged**